src/parse/stmt.rs

```diff
@@ -61,12 +62,15 @@ impl<I: Lexer> Parser<I> {
     /// | jump_statement
     /// ;
     ///
-    /// labeled_statement:
-    ///     identifier ':' statement
-    ///   | CASE constant_expr ':' statement
-    ///   | DEFAULT ':' statement
+    /// labeled_statement
+    /// : identifier ':' statement
+    /// | CASE constant_expr ':' statement
+    /// | DEFAULT ':' statement
+    /// ;
+    /// ```
     ///
     /// Result: whether there was an error in the program source
+    ///
     /// Option: empty semicolons still count as a statement (so case labels can work)
```

Comment:
lol this doesn't actually return `Option` anymore
```suggestion
```
And with that in mind I think there's no need to comment for `Result`.